pyband/transaction.py

```diff
@@ -57,11 +44,10 @@ def with_gas(self, gas: int) -> "Transaction":
     def with_memo(self, memo: str) -> "Transaction":
         if len(memo) > MAX_MEMO_CHARACTERS:
             raise ValueTooLargeError("memo is too large")
-
         self.memo = memo
         return self
 
-    def get_sign_data(self) -> bytes:
```

Comment:
You have merge `get_sign_data` and `get_tx_data` into one `get_tx_data` function. I think it would be better to not merge those functions.

- `get_sign_data` constructs `SignDoc` object and return as serialized bytes data.
- `get_tx_data` receive public key and signature, then construct `TxRaw` and return as serialized bytes data.

And let the user use their own way to sign the `SignDoc` data.